virtme/commands/run.py

```diff
@@ -1293,28 +1301,51 @@ def do_it() -> int:
     else:
         virtme_init_cmd = "virtme-init"
 
+    initsh = []
+    if args.systemd:
+        kernelargs.append("virtme_systemd=1")
+        kernelargs.append(f"console={arch.serial_console_args()[0]}")
```

Comment:
```suggestion
        kernelargs.append(" ".join([f"console={console}" for console in arch.serial_console_args()}")
```

Should have suggested it already in the beginning.

Reply:
Fixed in https://github.com/arighi/virtme-ng/pull/280/commits/3c7a573327e33912d2f87dd43acff2f30fe2931d